### DISCUSSION TITLE
Failed to create APIClient: Unable to set up TLS.

### DISCUSSION THREAD

**Discussion by HarryHoliday** [vercel/turborepo#7827] [Help] [answered]
### Summary

> turbo dev

  × Failed to create APIClient: Unable to set up TLS.
  ├─▶ Unable to set up TLS.
  ├─▶ builder error: The Trust Settings Record was corrupted.
  ╰─▶ The Trust Settings Record was corrupted.

**Comment by anthonyshew** [comment, marked as answer]
We believe we've got this solved in latest versions. Please open an issue if you're still seeing this problem.

**Reply by wanghaisheng** [reply]
windows 11 with latest version .same issue

**Reply by rickvdbroek** [reply]
@anthonyshew 

Still experiencing it on `MacOS 15.1`:
```bash
turbo 2.2.3

  × Failed to create APIClient: Unable to set up TLS.
  ├─▶ Unable to set up TLS.
  ├─▶ builder error: No such file or directory (os error 2)
  ╰─▶ No such file or directory (os error 2)

```

**Comment by hwondev** [comment]
@anthonyshew  thank you for your response

i have solved with this issue => https://github.com/vercel/turbo/issues/8330
